google-cloud-spanner/src/test/java/com/google/cloud/spanner/BackupTest.java

```diff
@@ -293,23 +293,40 @@ public void fromProto() {
   public void testEqualsAndHashCode() {
     final Backup backup1 = createBackup();
     final Backup backup2 = createBackup();
+    final Backup copyBackup1 = copyBackup();
 
     assertEquals(backup1, backup2);
     assertEquals(backup1.hashCode(), backup2.hashCode());
+    assertEquals(backup1.hashCode(), copyBackup1.hashCode());
   }
 
   private Backup createBackup() {
     com.google.spanner.admin.database.v1.Backup proto =
-        com.google.spanner.admin.database.v1.Backup.newBuilder()
-            .setName(NAME)
-            .setDatabase(DB)
-            .setExpireTime(
-                com.google.protobuf.Timestamp.newBuilder().setSeconds(1000L).setNanos(1000).build())
-            .setVersionTime(
-                com.google.protobuf.Timestamp.newBuilder().setSeconds(2000L).setNanos(2000).build())
-            .setEncryptionInfo(ENCRYPTION_INFO)
-            .setState(com.google.spanner.admin.database.v1.Backup.State.CREATING)
-            .build();
+            com.google.spanner.admin.database.v1.Backup.newBuilder()
```

Comment:
More of a question than an action item, but: Do you know why the formatter is changing the indentation here? Are you just using the normal `mvn com.coveo:fmt-maven-plugin:format` command, or is this a new format plugin?